coordinator/gscoordinator/utils.py

```diff
@@ -121,6 +122,16 @@
         GRAPHSCOPE_HOME, "interactive_engine", "bin", "giectl"
     )
 
+# JAVA SDK related CONSTANTS
+LLVM4JNI_HOME = os.environ.get("LLVM4JNI_HOME", "/opt/fastFFI/llvm4jni")
```

Comment:
Don't assuming the default path, as if we cannot continue if if not found.

If failed to failed, raise error *when users are trying to run java apps*.

Reply:
Now the logic for LLVM4JNI is: if CXX complier is clang AND LLVM4JNI_HOME found, we run llvm4jni.sh, otherwise we skip.